app/init.php

```diff
@@ -165,7 +165,7 @@ function($value) {
             PDONative::ATTR_ERRMODE => PDONative::ERRMODE_EXCEPTION,
         ]);
 
-    $pool = new PDOPool($config, 4096); // TODO: Investigate pool size
+    $pool = new PDOPool($config, 16384); // TODO: Investigate pool size
```

Comment:
Did we open an issue about this on the Swoole repo?

Reply:
Not yet, see previous comment.